lib/stsci/tools/tests/test_bitmask.py

```diff
@@ -0,0 +1,186 @@
+"""
+A module containing unit tests for the `bitmask` modue.
+
+:Authors: Mihai Cara (contact: help@stsci.edu)
+
+:License: `<http://www.stsci.edu/resources/software_hardware/pyraf/LICENSE>`_
```

Comment:
Please remove this. There is already a proper license at the root dir of this repo.